gaia-sdk-java/gaia-sdk-api/src/main/kotlin/gaia/sdk/api/identity/IdentityOp.kt

```diff
@@ -45,7 +52,22 @@ class IdentityOp(private val client: GaiaStreamClient) {
                     .toFlowable()
 
 
-    fun import(id: String?): Unit {
-        throw NotImplementedError("Implement identity import functionality is not yet implemented")
+    fun import(tenantId: String, identityName: String, content: File, override: Boolean = false, identityId: String? = null): Flowable<IdentityImportResponse> {
+        return Flowable.fromPublisher(DataRef("gaia://$tenantId/identities/", this.client).add(content.name, content, override))
+            .flatMap { dataRef ->
+                Flowable.fromPublisher(client.post(IdentityImportImpulse(dataRef.getUri(),
+                    tenantId,
+                    identityId ?: UUID.randomUUID().toString(),
+                    identityName,
+                    override), IdentityImportResponse::class.java, "/identity/import"))
+                .doOnError { reason ->
+                    val msg = "Importing Identity " + identityName + " failed: " + reason.message
+                    log.error(msg)
+                }
+                .map {
+                    it
```

Comment:
map {it} ist redundant - tut also nix weil it is ja eh schon das was vom Flowable daherkommt.
doOnError schluckt den Wert nicht, keine Sorge :-)